src/core/jit/store.rs

```diff
@@ -15,8 +23,9 @@ pub enum Data<A> {
     /// other of invocation and not the other of completion.
     // TODO: there could be multiple inside multiple in case of nested resolvers that are resolved
     // to lists
```

Comment:
```suggestion
```

With the new structure, the TODO seems to be handled, right?

Reply:
resolved separately in https://github.com/tailcallhq/tailcall/pull/2327